npm/ng-packs/packages/theme-shared/src/lib/components/internet-connection-status/internet-connection-status.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component } from '@angular/core';
+import { InternetConnectionService } from '../../../../../core/src/lib/services/internet-connection-service';
+
+
+
+@Component({
+  selector: 'abp-internet-status',
+  styleUrls: ['./internet-connection-status.scss'],
+  template: `
+    <div class="statusIcon" *ngIf="!internetConnection">
+      <i data-toggle="tooltip" title="The operation could not be performed. Your internet connection is not available at the moment." data-placement="top"  class="fa fa-circle text-blinking Blink"></i>
+    </div>
+  `,
+})
+export class InternetConnectionStatusComponent{
+  internetConnection: boolean;
+  constructor(private internetConnectionService:InternetConnectionService){

```

Comment:
- use [inject](https://angular.io/api/core/inject) function
- Don't need to subscribe use `async` pipe for internetConnection variable